### ISSUE TITLE
jeet getting replaced by lost?

### ISSUE THREAD

**Issue by 490a** [mquandalle/meteor-stylus#8]
it looks like maybe jeet is getting replaced by lost instead of going to version 7 - v7 is waiting on corysimmons' approval, but he's working on lost:

https://github.com/corysimmons/lost

will you make the switch?

**Comment by mquandalle** [comment]
Since you have to explicitly `@import 'jeet'` to use it, we don't need to force users to do the switch. Instead we could add lost as one more import-able stylus library and users will chose what they want to use.

**Comment by cryptoquick** [comment]
Yeah, there's no switch involved; instead, there's only the fact that we should support it if people would like to use it. I can take care of this sometime soon.

**Comment by mquandalle** [comment]
As explained in the above commit and in https://github.com/corysimmons/lost/issues/102 lost v5.1.6 is the last `lost` version with direct stylus support.

**Comment by priezz** [comment]
Can we use `lost` via `poststylus` (https://github.com/seaneking/poststylus)? Including poststylus into the package should give additional benefits.

**Comment by Tevinthuku** [comment]
Yes we can
